### ISSUE TITLE
swagger-ui is running fine locally..but when deployed on server its giving 404 error

### ISSUE THREAD

**Issue by rrraaa1** [springfox/springfox#3315]
Please take the time to search the repository, if your question has already been asked or answered.

- [x] What version of the library are you using? Is it the latest version? The latest released version is [ ![Download](https://api.bintray.com/packages/springfox/maven-repo/springfox/images/download.svg) ](https://bintray.com/springfox/maven-repo/springfox/_latestVersion)

What kind of issue is this?

 - [x] Question. Is this a question about how to do a certain thing? 

 - [ ] Bug report. If you’ve found a bug, spend the time to write a failing test. Bugs with tests or 
       steps to reproduce get fixed faster. Here’s an example: https://gist.github.com/swankjesse/6608b4713ad80988cdc9
    - [ ] spring xml/java config that is relevant
    - [ ] springfox specific configuration if it makes sense
    - [ ] include any output you've received; logs, json snippets etc.; and what the expected output should be
    - [ ] if you have a repo that demonstrates the issue for bonus points! See [this example](https://github.com/springfox/springfox/issues/494) 

 - [ ] Feature Request. Start by telling us what problem you’re trying to solve. Often a solution
       already exists! Don’t send pull requests to implement new features without first getting our
       support. Sometimes we leave features out on purpose to keep the project small.

Please do consider ***starring*** this repository if you do like this project and find it useful.

**Comment by dilipkrish** [comment]
This should now be fixed in 3.0.0. the new url is `/swagger-ui/index.html` or `/swagger-ui/` instead of `/swagger-ui.html`. Please refer the docs on how to upgrade

**Comment by aks103204** [comment]
@dilipkrish Hi sir, i have the same situation, it works locally(http://localhost:8080/swagger-ui/index.html#/), but not works in server. There are my config
```
user  nginx;
worker_processes  auto;

error_log  /var/log/nginx/error.log notice;
pid        /var/run/nginx.pid;


events {
    worker_connections  1024;
}


http {
    include       /etc/nginx/mime.types;
    default_type  application/octet-stream;
    access_log  /var/log/nginx/access.log;

    sendfile        on;

    keepalive_timeout  65;
	
server {
     listen 80;
location /swagger/ {
    	    proxy_pass http://172.17.0.3:8080/swagger-ui/index.html;
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         
     
}

}

}
```  my server ip in the docker container is 172.17.0.3, port 8080. Could you help me for this, thanks.
